charts/MapLegend.tsx

```diff
@@ -708,7 +708,6 @@ export class MapLegendView extends React.Component<MapLegendViewProps> {
                         onMouseLeave={onMouseLeave}
                     />
                 )}
-                />}
```

Comment:
This is not directly related to this PR, but this is a bogus closing tag/brace.
I'm surprised no linter caught this.

Reply:
Ha, I'm surprised too! It seems it's valid JSX to add as many of those closing tags you want.

Reply:
Yeah, they were just added to the the DOM, so they were considered HTML at that point. Thankfully, it was in a place where the browser just ignored them.